CONTRIBUTING.md

```diff
@@ -95,6 +95,21 @@ Then add a title and description to your pull request that explains your contrib
 <img width="882" alt="pullr" src="https://user-images.githubusercontent.com/44089458/86092345-fed8c280-baca-11ea-9a04-95991d9c60d2.PNG">  
 - Voila! Your Pull Request has been submitted and will be reviewed by the moderators and merged.🥳
 
+### Note : Do not add images, rather 👇
+
+- We recently have removed all the images and screenshots from our repository and linked them to markdown files.
+    
+    ## How to do that? ##

```

Comment:
You are using an h2 after h3.

The Markdown Styling will report that as an error. Also, different types of Syntax shouldn't be used.

So either use h4 or make the line bold.